packages/nextjs/components/scaffold-stark/Address.tsx

```diff
@@ -60,6 +65,19 @@ export const Address = ({
     checkSumAddress || address || "",
   );
 
+  const isValidHexAddress = (value: string): boolean => {
+    if (value.toLowerCase() === "0x") {
+      value = "0x0";
+    }
+
+    if (value.toLowerCase() === "0x0x0") {
+      return false;
+    }
```

Comment:
`0x0x0` is just an example, what if the address is like `0x0x141240x` ? this is supposed to be invalid. Have to account for all the cases as well.